test/e2e_kubeadm/bootstrap_token_test.go

```diff
@@ -34,6 +35,21 @@ const (
 	bootstrapTokensCSRAutoApprovalClusterRoleName    = "system:certificates.k8s.io:certificatesigningrequests:nodeclient"
 )
 
+var (
+	bootstrapTokensAllowPostCSR = &authv1.ResourceAttributes{
+		Resource: "certificatesigningrequests",
+		Group:    "certificates.k8s.io",
+		Verb:     "create",
```

Comment:
the `system:node-bootstrapper` has other important verbs, nut just `create`:
```
  - create
  - get
  - list
  - watch
```

without these TLS bootstrap will fail.

i'm not convinced that we need to check them, because these bindings are not managed by kubeadm source code.

/hold 
/cc @fabriziopandini 
deferring to @fabriziopandini as the author of the TODO line.

Reply:
I got your idea. Looking forward Fabrizio's reply.